documentation/book/proc-configuring-maintenance-time-windows.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// assembly-maintenance-time-windows.adoc
+
+[id='proc-configuring-maintenance-time-windows-{context}']
+= Configuring maintenance time windows
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Edit the `maintenanceTimeWindows` property in the `Kafka` resource.
```

Comment:
Add an additional resources section to the end of the assembly. These should be linked to the appropriate procedures.

**Additional resources**
- Performing a rolling update of a Kafka cluster
- Performing a rolling update of a Zookeeper cluster.